### DISCUSSION TITLE
Luminance constants too precise?

### DISCUSSION THREAD

**Discussion by anikom15** [ImageMagick/ImageMagick#5693] [Development] [answered]
I noticed this in the documentation:

luminance: pixel luminance; equivalent to 0.212656*red + 0.715158*green + 0.072186*blue

Aren't these constants too precise? Both sRGB and BT.709 use 0.2126, 0.7152, and 0.0722. I'm sure in practice, it doesn't make much of a difference, if any, for 8-bit images, but shouldn't ImageMagick follow the standards exactly? Or am I misinterpreting these numbers in some way...

**Comment by urban-warrior** [comment, marked as answer]
For the interchange  with other packages, picking the standard might be beneficial, however, for color transformation pipelines, derived matrices are preferred to reduce rounding errors.  In practice, there is little difference.  Note, our  color conversion math comes from http://www.brucelindbloom.com/index.html?WorkingSpaceInfo.html.
